src/components/profile/profile.pw.tsx

```diff
@@ -0,0 +1,187 @@
+    [PROFILE_SIZES[3], 54],
+    [PROFILE_SIZES[4], 70],
+    [PROFILE_SIZES[5], 102],
+    [PROFILE_SIZES[6], 126],
+  ] as [ProfileProps["size"], number][]).forEach(([size, heightAndWidth]) => {
+    test(`should check when size prop is passed as ${size}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent size={size} />);
+
+      await expect(initialPreview(page)).toHaveCSS(
+        "height",
+        `${heightAndWidth}px`
+      );
+      await expect(initialPreview(page)).toHaveCSS(
+        "width",
+        `${heightAndWidth}px`
+      );
+    });
+  });
+
+  [
+    ["Dan Jin", "DJ"],
+    ["Sid Ford", "SF"],
+  ].forEach(([name, passInitials]) => {
+    test(`should check name initials when name prop is passed as ${name}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent initials={passInitials} name={name} />);
+
+      await expect(initialPreview(page)).toHaveCSS("height", "38px");
+      await expect(initialPreview(page)).toHaveCSS("width", "38px");
+    });
+  });
+
+  imageURLs.forEach((url) => {
+    test(`should check when src prop is passed as ${url}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent src={url} />);
+
+      await expect(portraitImage(page)).toHaveCSS("height", "40px");
+      await expect(portraitImage(page)).toHaveCSS("width", "40px");
+      await expect(portraitImage(page)).toHaveAttribute("src", url);
+    });
+  });
+});
+
+test.describe("should check Accessibility tests for Profile component", () => {
+  test(`should pass accessibility tests for className prop`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ProfileComponent className="profile-classnames" />);
+
+    await checkAccessibility(page);
+  });
+
+  testData.forEach((email) => {
+    test(`should pass accessibility tests for email prop set as ${email}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent email={email} src={testImage} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((name) => {
+    test(`should pass accessibility tests for name prop set as ${name}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent name={name} src={testImage} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  (["XS", "S", "M", "ML", "L", "XL", "XXL"] as const).forEach((size) => {
+    test(`should pass accessibility tests for size prop set as ${size}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent size={size} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  [
+    ["Dan Jin", "DJ"],
+    ["Sid Ford", "SF"],
+  ].forEach(([name, passInitials]) => {
+    test(`should pass accessibility tests for initials when name prop is set as ${name}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ProfileComponent initials={passInitials} name={name} />);
```

Comment:
**suggestion** in this specific scenario I think we want to be testing without initials being passed, so remove `initials={passInitials}`

Reply:
I can see this hasn't been addressed, is this because the test is failing if we don't specifically pass the initials prop?